### ISSUE TITLE
Only show a limited and trimmed range in upper view for long lines

### ISSUE THREAD

**Issue by pintassilgo** [kvichans/cuda_find_in_files4#80]
Sometimes I need to search in files with long lines, then FiF lists results showing the entire line for each match. This defeats the purpose of the plugin for me, which is a list summarizing each result.

1. I suggest a way/option so that results are listed one per line, without wrapping long lines, preferably with the searched word/string centered to provide better context.

2. Other improvement in this regard would be an option to trim indentation in this upper view, because this blank space on left would be better used to display more text for the user to have more context.

3. Moved to https://github.com/kvichans/cuda_find_in_files4/issues/81.

**Comment by Alexey-T** [comment]
>(please let me know if it's better to make a new issue for each thing)

New issue for that - will be better.

**Comment by Alexey-T** [comment]
>I suggest a way/option so that results are listed one per line

FIF4 in the '=' button has some options. did you try them all? see the bottom part of '=' menu:

- show relative paths
- results tree >>

**Comment by Alexey-T** [comment]
Can you give the patch with the option?

**Comment by Alexey-T** [comment]
FIF4 last is 4.8.15, but your screenshot - 4.8.12.

I will look why word-wrap is On for you. for me - it is Off. check it on last version.

**Comment by Alexey-T** [comment]
Yes, wrap-mode is got from user.json. It must be easy to fix it, I will try.

**Comment by Alexey-T** [comment]
do you suggest to hardcode wrap=Off for both editors (upper/lower)? for one of them?

**Comment by Alexey-T** [comment]
Fix made, only in Git. the proper release will be much later, after next fixes.

**Comment by Alexey-T** [comment]
If this issue is solved, pls close. other 2 issues should be in separate GH issues.
